pkg/scheduler/preemption/preemption.go

```diff
@@ -64,7 +64,7 @@ func (p *Preemptor) OverrideApply(f func(context.Context, *kueue.Workload) error
 }
 
 func candidatesOnlyFromQueue(candidates []*workload.Info, clusterQueue string) []*workload.Info {
-	result := make([]*workload.Info, 0)
+	result := make([]*workload.Info, 0, len(candidates))
```

Comment:
nit: I would suggest to revert unrelated changes

Reply:
also, all the workloads might be from a different CQ.

So it could be better to do `var result []*workload.Info`

Reply:
It's clear here that the length wouldn't greater than len(candidates).